<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageWidget</class>
 <widget class="QWidget" name="ImageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>568</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="KComboBox" name="cbGraphType">
         <property name="toolTip">
          <string>Vertical position relative to label's parent</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lGeometry">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Geometry:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lPositionX">
         <property name="toolTip">
          <string>Horizontal position relative to label's parent</string>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KComboBox" name="cbPositionX">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Horizontal position relative to label's parent</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="sbPositionX">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Horizontal position relative to label's parent</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KComboBox" name="cbPositionY">
         <property name="toolTip">
          <string>Vertical position relative to label's parent</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lPositionY">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Vertical position relative to label's parent</string>
         </property>
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Reference Point</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lRotation">
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="sbPositionY">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Vertical position relative to label's parent</string>
         </property>
         <property name="suffix">
          <string> cm</string>
         </property>
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="sbRotation">
         <property name="suffix">
          <string>Â°</string>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lPoisitionX1">
         <property name="text">
          <string>x[1]</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QDoubleSpinBox" name="sbPoisitionX3"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lGraphType">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Graph Type</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="sbPoisitionX1">
         <property name="decimals">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="lPoisitionY1">
         <property name="text">
          <string>y[1]</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QDoubleSpinBox" name="sbPoisitionY1">
         <property name="maximum">
          <double>99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lPoisitionX2">
         <property name="text">
          <string>x[2]</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="sbPoisitionX2"/>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="lPoisitionY2">
         <property name="text">
          <string>y[2]</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QDoubleSpinBox" name="sbPoisitionY2"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lPoisitionX3">
         <property name="text">
          <string>x[3]</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLabel" name="lPoisitionY3">
         <property name="text">
          <string>y[3]</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QDoubleSpinBox" name="sbPoisitionY3"/>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="chbVisible">
         <property name="text">
          <string>visible</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>231</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tSymbol">
      <attribute name="title">
       <string>Symbol</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8"/>
     </widget>
     <widget class="QWidget" name="tEdit">
      <attribute name="title">
       <string>Edit</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
